_includes/card.html

```diff
@@ -8,21 +8,22 @@
   {% assign wds_grid_class = 'usa-width-one-third' %}
 {% endif %}
 
-{% assign project_type = include.project_type | default: 'none' %}
-
-<article class="card {{ wds_grid_class }}" data-bucket="{{ project_type }}">
-  <a class="card-link" href="{{ card_link }}">
-    <div role="img" title="{{ include.image_alt }}" class="card-image-bg" style="background-image: url({{ site.baseurl }}{{ include.image_src }});">
-    </div>
+<article class="card {{ wds_grid_class }}">
+  <div class="card-link" onclick='window.location = "{{ include.link }}";' tabindex="-1">
+    <div role="img"
+         title="{{ include.image_alt }}"
+         class="card-image-bg"
+         style="background-image: url({{ site.baseurl }}{{ include.image_src }});"></div>
     <div class="card-banner">
       <p class="card-tag">{{ include.agency }}</p>
-      <h3 class="h4">{{ include.tagline }}</h3>
+      <a href="{{ card_link }}" tabindex="-1" class="card-link-tagline"><h3 class="h4">{{ include.tagline }}</h3></a>
       <p class="card-description">{{ include.description }}</p>
-      <span class="link-arrow-right">
+      <a href="{{ card_link }}" class="link-arrow-right" tabindex="0">
         Read more
+        <span class="usa-sr-only">about {{ include.tagline }}</span>
```

Comment:
I tested this in VoiceOver and it's reading it backwards ("about..." is first, then "Read more"):
<img width="1188" alt="screen shot 2017-01-23 at 3 19 20 pm" src="https://cloud.githubusercontent.com/assets/5249443/22227357/bd25b01e-e17f-11e6-9c04-bf29f45423f7.png">

Does "Read more" have to be in a `span` to be read first, since the other text is in a span? /cc @nickbristow